project/AppDependencies.scala

```diff
@@ -19,27 +19,25 @@ import sbt.*
 
 object AppDependencies {
 
-  val hmrcMongoVersion = "1.3.0"
-  val bootstrapVersion = "7.23.0"
+  val hmrcMongoVersion = "1.8.0"
```

Comment:
The dependency updates tool reckons we can go to 1.9.0 now.

Reply:
Moved to `1.9.0`